cli/Makefile

```diff
@@ -4,4 +4,4 @@ lint:
 
 .PHONY: test
 test:
-	go test ./...
\ No newline at end of file
+	go test ./...
```

Comment:
This is a stub change to trigger a release in the monorepo (as we detect changes based on paths in the repo)